client/src/app/site/motions/components/motion-detail/motion-detail.component.html

```diff
@@ -268,11 +268,29 @@ <h3 translate>Voting</h3>
             </button>
         </div>
 
+        <!-- Selecting statute paragraphs for amendment -->
+        <div class="statute-amendment-selector" *ngIf="editMotion && statuteParagraphs.length > 0">
```

Comment:
Looks out of place here and moves the text and reason around.
I would move it into the meta data block and perhaps just use a `mat-select` without a checkbox